fixit/rules/use_assert_is_not_none.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.helpers import ensure_type
+
+from fixit.common.base import CstLintRule
+from fixit.common.utils import InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+class UseAssertIsNotNoneRule(CstLintRule):
+    """
+    Discourages use of ``assertTrue(x is not None)`` and ``assertFalse(x is not None)`` as it is deprecated (https://docs.python.org/3.8/library/unittest.html#deprecated-aliases).
+    Use ``assertIsNotNone(x)`` and ``assertIsNone(x)``) instead.
+
+    """
+
+    MESSAGE: str = (
+        '"assertTrue" and "assertFalse" are deprecated. Use "assertIsNotNone" and "assertIsNone" instead.\n'
+        + "See https://docs.python.org/3.8/library/unittest.html#deprecated-aliases"
+    )
+
+    VALID = [
+        Valid("self.assertTrue(not x is None)"),
+        Valid("self.assertTrue(x)"),
+        Valid("self.assertFalse(not x is None)"),
+        Valid("self.assertFalse(x)"),
```

Comment:
`not x is None` is the same as `x is not None`
It can actually be also handled by this rule to provide an autofix.
Let's also add matcher pattern for it.

Use assertTrue/assertFalse as an example may cause confusion to users since they are deprecated.
(The Valid and Invalid cases are generated as examples in the documentation like this)
https://fixit.readthedocs.io/en/latest/rules/ComparePrimitivesByEqualRule.html
Let's use the suggested `assertIsNotNone` and `assertIsNone` as Valid case examples.

Reply:
Going to add a few more ensure_types to see if I can get the typecheck to pass on Ubuntu + 3.7